blogposts/2021/dev-tool-time-mitchell-hashimoto.md

```diff
@@ -37,39 +37,43 @@ Mitchell also has a stack of [Raspberry Pi](https://www.raspberrypi.org/) comput
+Mitchell uses his iPad and Apple Pencil for a mix of personal and professional work. As a hobbyist pilot, he plots and checks flight paths. As an engineer, he uses the iPad for reading and reviewing code and long-form documents, such as RFCs, and writing handwritten notes with the Apple Pencil and [Goodnotes](https://www.goodnotes.com/). Goodnotes makes it easy to move text around, and reorganize content visually. When he's ready for long-form typing or document creation, Mitchell transfers his notes to his computer with Goodnotes for MacOS and iCloud before drafting in Google Docs, GitHub, or the appropriate app. This setup frees Mitchell from his desk, letting him do code reviews outside.
 
 ## [NixOS](https://nixos.org/)
 
 > "I think one thing that always scared me every single year was the MacOS release and like, what would that break?"
 
+Operating system and other software updates regularly break development environments. Mitchell avoids this recurring problem using NixOS in a VM running with [VMware Fusion](https://www.vmware.com/products/fusion.html).
+
 > "My biggest criticism about Nix is that their learning is like a sheer cliff. It's the hardest learning curve of probably in tech I've ever learned."
 
-Operating system and other software updates regularly break development environments. Mitchell avoids this recurring problem using NixOS in a VM running with [VMWare Fusion](https://www.vmware.com/products/fusion.html). Despite it's difficult learning curve, NixOS makes the entire development environment reproducible and easy to change, knowing that you can always rollback to previous versions.
+ Despite its difficult learning curve, NixOS makes the entire development environment reproducible and easy to change, knowing that you can always roll back to previous versions.
 
 Within the NixOS VM, two other tools help Mitchell set and use packages and environment variables just for one session:
 
 - [nix-shell](https://nixos.org/manual/nix/stable/#sec-nix-shell) enables a single-session shell with specific dependencies
-- [direnv](direnv) provides a consistent approach for setting and accessing environment variables based on directory
+- [direnv](https://direnv.net/) provides a consistent approach for setting and accessing environment variables based on directory
 
 ## [Neovim](https://neovim.io/)
 
-> "I just don't feel like I personally need an IDE experience."
-
-> "I don't like things that write code for me; I want to do it myself."
+> "I used to be an Emacs user ... and then I joined this other company and I was harassed on a daily basis about my Emacs use because I was the only Emacs user in the entire company ... I was the butt of every joke! I got so fed up with it I decided to use Vim for one month and I never switched back."
```

Comment:
This felt like a more colourful quote so I led with this, and combined the original two pull quotes with ellipses, let me know if this works for you!